gcp-deployer/machine_setup_configs.yaml

```diff
@@ -112,6 +113,21 @@ items:
       modprobe br_netfilter
 
       kubeadm init --config /etc/kubernetes/kubeadm_config.yaml
+
+      # add default storage class
+      cat > /etc/kubernetes/default-storage-class.yaml <<EOF
+      kind: StorageClass
+      apiVersion: storage.k8s.io/v1
+      metadata:
+        name: standard
+        annotations:
+          storageclass.kubernetes.io/is-default-class: "true"
+      provisioner: kubernetes.io/gce-pd
+      parameters:
+        type: pd-standard
+      EOF
+      kubectl apply --kubeconfig /etc/kubernetes/admin.conf -f /etc/kubernetes/default-storage-class.yaml
+
```

Comment:
Is this the best place to install this? The API is running at this point, and we should be able to install this from anywhere. We don't want to do this on every master create (in a multi-master scenario). Also, would be nice if we could do this in a place that isn't a bash install script.

It's fine for now, but we will likely want to move away from this.